### ISSUE TITLE
Clear out unused dependencies

### ISSUE THREAD

**Issue by Rexogamer** [DTel-HQ/dtel#288]
In [our current `package.json`](https://github.com/DTel-HQ/dtel/blob/ad8ea883d476e2f2ade22d2c2b542d2ae23763dd/package.json), we currently have *twenty-three* dependencies. While that's certainly not a huge amount, I suspect at least some of these are either entirely unused or replaceable.

**Comment by Mitchell3514** [comment]
Unused deps:
- re2
- zucc
- require-reload

**Comment by Rexogamer** [comment]
- re2 was missing when I tried to build the bot before
- zucc seems fine to remove, same with require-reload
- will look into replacing randomstring

**Comment by turret-dev** [comment]
Currently in the state of the repository in `development`, testing the bot to see if there are any issues is unavailable, for I believe that the files are mixed between typescript & javascript at the time, and without a valid `tsconfig.json` set up, and without the rest of the files (hopefully) being remade in typescript, node won't doesn't like the repository at the time. This one will be on hold until we get typescript setup

TLDR; dev branch is messed up rn & we cant test what would happen if we removed each dependency, so im gonna put this one on hold for now

**Comment by SunburntRock89** [comment]
@turret-dev Which files are in TS?

**Comment by SunburntRock89** [comment]
ah wait right I see what you mean now, someone was using import statements which JS isn't happy about
